x-pack/solutions/observability/plugins/observability_ai_assistant_app/scripts/evaluation/get_service_urls.ts

```diff
@@ -23,7 +23,7 @@ async function discoverAuth(parsedTarget: Url, log: ToolingLog) {
       const response = await fetch(url);
       status = response.status;
     } catch (err) {
-      log.debug(`${url} resulted in ${err.message}`);
+      log.error(`${url} resulted in ${err.message}`);
```

Comment:
I think `debug` here is fine honestly. It's just trying several credentials until it finds one that works. If one fails, it might be expected, so I don't think it warrants `error`.